edxval/api.py

```diff
@@ -182,6 +144,53 @@ def update_video_status(edx_video_id, status):
     video.save()
 
 
+def get_course_video_image_url(course_id=None, edx_video_id=None, video_image=None):
+    """
+    Returns course video image url.
+
+    Arguments:
+        course_id: ID of course
+        edx_video_id: ID of the video
+        video_image: VideoImage instance
+
+    Returns:
+        course video image url
+    """
+    storage = get_video_image_storage()
+
+    try:
+        if video_image is None:
+            video_image = CourseVideo.objects.get(course_id=course_id, video__edx_video_id=edx_video_id).video_image
+    except ObjectDoesNotExist:
```

Comment:
Infact we need to catch two exceptions: 1 for CourseVideo.DoesNotExist and 1 for course_video.video_image.